src/main/resources/db/migration/V1.1_001__message_received_precision.sql

```diff
@@ -0,0 +1,4 @@
+
+-- github issue 8 / add millisecond resolution
+alter table message modify received_date timestamp(3) not null;
```

Comment:
Yeah - Do you think it's worth rigging the integration tests to run against a suite of supported DBs (H2, MariaDB etc)?